CMakeLists.txt

```diff
@@ -36,6 +36,10 @@ elseif (CMAKE_SYSTEM_NAME STREQUAL "Darwin")
 	add_definitions(-DRW_OSX)
 elseif(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
 	add_definitions(-DRW_FREEBSD)
+elseif(CMAKE_SYSTEM_NAME STREQUAL "OpenBSD")
+	add_definitions(-DRW_OPENBSD)
+	include_directories("/usr/X11R6/include")
```

Comment:
This path is constant on OpenBSD. [Xenocara](https://en.wikipedia.org/wiki/Xenocara) is OpenBSD's version of the x.org server. It won't move around on this platform :)